### ISSUE TITLE
Session is already started error.

### ISSUE THREAD

**Issue by creativefctr** [gburtini/PHP-Brute-Force-Defense#1]
Hi,
You have added session_start() in the constructor, that causes a notice error when the application has already started the session itself. It's better to add a check for session_started in the constructor:

``` PHP
if (session_status() == PHP_SESSION_NONE) {
    session_start();
}
```

Thanks

**Comment by gburtini** [comment]
We check this -- at least in the master branch. It is not a nice check (session_status is much nicer than checking for an empty session_id like we currently do), but let me know if you're working on the master branch and I'm just misunderstanding you.

G.

**Comment by creativefctr** [comment]
@gburtini 
I installed it from composer with:

``` JSON
"gburtini/bfd": "*"
```

changing \* to "dev-master" fixed it. But I think it would be helpful to mention this in your read me since mine did not get updated because of the *

**Comment by gburtini** [comment]
Ah, that's what it is. I have fixed it by pushing a 0.0.6 tag.
